tests/business-logic/User.test.ts

```diff
@@ -0,0 +1,83 @@
+import UserEntity from "@business-logic/User";
+
+import { hashPassword } from "@helpers/auth";
+import prisma from "@helpers/prisma";
+import { teardown } from "@helpers/tests/teardown";
+
+let passwordHash: string;
+const email = "test@nueno.com";
+
+describe("User Entity", () => {
+  beforeEach(async () => {
+    await teardown();
+    passwordHash = await hashPassword("test@123.");
+  });
+
+  describe("#CreateNewUser", () => {
+    it("should create new user", async () => {
+      const body = {
+        name: "Chris Rock",
+        email,
+        password: passwordHash,
+      };
+      const userEntity = new UserEntity();
+      const result = await userEntity.create(body);
+      expect(result.message).toBe("User created");
+    }, 40000);
+    it("should not create user if email already exists", async () => {
+      const body = {
+        name: "Chris Rock",
+        email,
+        password: passwordHash,
+      };
+      const entity = new UserEntity();
+      await entity.create(body);
+      await expect(async () => {
+        await entity.create(body);
+      }).rejects.toThrowError("Email address is already registered.");
+    }, 40000);
+    it("Should not create user when email is invalid", async () => {
```

Comment:
just a small thing and personal preference. can you add an empty paragraph between the `it('...')`. 

I also prefer paragraphs in between the blocks

```javascript
  it("should return an existing user", async () => {
    const company = await prisma.company.create({
      data: {
        name: "Academy Records",
      },
    });

    const user = await prisma.user.create({
      data: {
        name: "Chris Rock",
        email: "test2@nueno.com",
        password: passwordHash,
        companyId: company.id,
      },
    });

    const userEntity = new UserEntity();
    const result = await userEntity.find(user.id);

    expect(result?.name).toBe(user.name);
  }, 4000);
```